vault/core_util.go

```diff
@@ -199,3 +201,6 @@ func (c *Core) MissingRequiredState(raw []string, perfStandby bool) bool {
 func DiagnoseCheckLicense(ctx context.Context, vaultCore *Core, coreConfig CoreConfig, generate bool) (bool, []string) {
 	return false, nil
 }
+
+func (c *Core) reloadSealsEnt(secureRandomReader io.Reader, sealAccess seal.Access, logger hclog.Logger) {
```

Comment:
Maybe you need to use stub maker? The documentation can be found at https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/2295365741/The+Vault+Development+Cycle#Ent-code-invoked-from-OSS